pageserver/src/tenant/storage_layer/inmemory_layer.rs

```diff
@@ -825,7 +824,7 @@ impl InMemoryLayer {
                             len,
                             will_init,
                         } = entry;
-                        let buf = Bytes::slice(&file_contents, pos as usize..(pos + len) as usize);
+                        let buf = file_contents.slice(pos as usize, (pos + len) as usize);
```

Comment:
a little bit weird that `slice` takes start+end instead of start+len, but fine 🤣

Reply:
Yea, I think will just followed the https://docs.rs/bytes/latest/bytes/struct.Bytes.html#method.slice convention: ad44e11a6937659d7dc7bd1785570a0d0a894534.